EasyVCR.Tests/FakeDataService.cs

```diff
@@ -1,31 +1,20 @@
 using System;
-using System.Collections.Generic;
 using System.Net.Http;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
 
+// ReSharper disable InconsistentNaming
```

Comment:
Question: What InconsistentNaming was it complaining about here?

Reply:
`IPAddress` vs `IpAddress`

Reply:
Can we address its concerns? I suppose I don't understand why we need to disable it.